scripts/prepare_sector_network.py

```diff
@@ -202,10 +203,10 @@ def build_carbon_budget(o, fn):
 
     countries = n.buses.country.dropna().unique()
 
-    e_1990 = co2_emissions_year(countries, opts, year=1990)
+    e_1990 = co2_emissions_year(countries, snakemake.input.eurostat, opts, year=1990)
```

Comment:
Could move any snakemake up to function arguments of build_carbon_budget() as well.

Reply:
yes, agrree.